src/Microsoft.Azure.ServiceBus/Constants.cs

```diff
@@ -40,5 +40,7 @@ static class Constants
         public static readonly int DefaultClientPumpPrefetchCount = 5;
 
         public static readonly TimeSpan NoMessageBackoffTimeSpan = TimeSpan.FromSeconds(5);
+
+        public static readonly TimeSpan MessageReceiverStartPumpInitialReceiveTimeout = TimeSpan.FromSeconds(1);
```

Comment:
I tried that with `TimeSpan.Zero` and it wasn't the case, it was take much longer than 10 seconds. I also tried milliseconds, same effect.